loops-arrays-objects.js

```diff
@@ -75,16 +107,23 @@ const NOVIEmployees = [
 //  ];
 // ==========================================
 
+for (let i = 0; i < NOVIEmployees.length; i++) {
 
+    NOVIEmployees[i]['email'] = ('' + NOVIEmployees[i].firstName + '.' + NOVIEmployees[i].lastName + '@novi.nl')
```

Comment:
```suggestion
    NOVIEmployees[i].email = NOVIEmployees[i].firstName + '.' + NOVIEmployees[i].lastName + '@novi.nl'
```

Een alternatieve manier (wat ik zelf makkelijker te lezen vind):
```suggestion
    NOVIEmployees[i]['email'] = `${NOVIEmployees[i].firstName}.${NOVIEmployees[i].lastName}@novi.nl`;
```